### PR TITLE
updated cudf-pandas content

### PR DESCRIPTION
**Pull request by exactlyallan**
Removed Beta and new mentions, added updated benchmark.

### CODE REVIEW

File: layouts/page/cudf-pandas.html [hunk 1]
```diff
@@ -90,7 +90,9 @@ <h2><i class="fa-light fa-square-info"></i> How to Use It </h2>
         <div class="col-md-6 py-3">
           <h2><i class="fa-light fa-gauge-high"></i> 150x Faster, Zero Code Change </h2>
           <canvas id="perfchart" width="500" height="375"></canvas>
-          <p class="fs-7 fw-lighter text-center">* Grace Hopper vs Intel Xeon Platinum 8480CL CPU <a
+          <p class="fs-7 fw-lighter text-center">* Standard DuckDB Data Benchmark (5GB) <br>
+            GPU: NVIDIA Grace Hopper, CPU: Intel® Xeon® Platinum 8480C <br>
+            pandas v2.2, RAPIDS cuDF v23.10 <a
```
Comment (bdice) [inline layouts/page/cudf-pandas.html:95]: Is there a way to get data for a newer cuDF version? The data from pandas 2.2 (published January 19, 2024) would be newer than the corresponding cuDF data from 23.10.

File: layouts/index.html [hunk 2]
```diff
@@ -219,8 +214,8 @@ <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i> RAPIDS Release Sel
                          detailed installation steps, and information about supported platforms. </p>
                 </div>
                 <div class="col-md-6 py-3">
-                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Test Drive the new cuDF</h2>
-                    <p>Try out cuDF's new pandas Accelerator Mode, with a free required account, right now by <a
+                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Test Drive cuDF</h2>
+                    <p>Try out cuDF's pandas Accelerator Mode, with a free required account, right now by <a
```
Comment (bdice) [inline layouts/index.html:218]: Avoid the possessive here. I left two possible suggestions -- take your favorite one.

I think the "free required account" piece of the first one is a bit awkward, so I'd lean towards the second suggestion.
```suggestion
                    <p>Try out cuDF pandas Accelerator Mode, with a free required account, right now by <a
```
```suggestion
                    <p>Try cuDF pandas Accelerator Mode in a free GPU-enabled notebook environment by <a
```

File: layouts/index.html [hunk 3]
```diff
@@ -219,8 +214,8 @@ <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i> RAPIDS Release Sel
...
                             href="https://nvda.ws/rapids-cudf" target="_blank">Launching Google CoLab <i
```
Comment (bdice) [inline layouts/index.html:219]:
```suggestion
                            href="https://nvda.ws/rapids-cudf" target="_blank">launching Google Colab <i
```

File: layouts/index.html [hunk 4]
```diff
@@ -105,11 +100,11 @@ <h2><i class="fa-light fa-chart-network"></i> Open Source Ecosystem </h2>
 
             <div class="row has-background-rapids-image ">
                 <div class="col-md-12 py-5">
-                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> NEW: Pandas Accelerator Mode
+                    <h2 class="text-white"><i class="fa-light fa-gauge-circle-bolt"></i> Pandas Accelerator Mode
                         for cuDF </h2>
-                    <p class="text-white">Use cuDF's new pandas Accelerator Mode to speed up pandas workflows with zero
+                    <p class="text-white">Use cuDF's pandas Accelerator Mode to speed up pandas workflows with zero
```
Comment (bdice) [inline layouts/index.html:105]: Avoid the possessive here.
```suggestion
                    <p class="text-white">Use cuDF pandas Accelerator Mode to speed up pandas workflows with zero
```

File: layouts/page/cudf-pandas.html (hunk 1 above)
Comment (exactlyallan) [reply layouts/page/cudf-pandas.html:95]: Eventually - this is the latest we have.

**Review by bdice — APPROVED** [review]

**Pull request merged**